Imagine/Filter/RelativeResize.php

```diff
@@ -45,6 +45,8 @@ public function __construct($method, $parameter)
 
     /**
      * {@inheritdoc}
+     *
+     * @return ImageInterface
```

Comment:
why change it exactly for this filter?

i think for version 3.x we want to declare return types on the FilterInterface, but for version 2.x i would leave things alone. the interface already defines the return type annotation.

Reply:
To be honest, this one was automatically added by 

`SYMFONY_PATCH_TYPE_DECLARATIONS=force=phpdoc ./vendor/bin/patch-type-declarations`

I'll revert it

Reply:
thanks